scripts/mkdist.js

```diff
@@ -14,43 +14,40 @@ function mkdist (opts, cb) {
 
   var dist = opts.dist
   var ver = dist.version
-  if (ver.match(/^v/)) ver = ver.substring(1)
```

Comment:
Sure. We don't have versioning nor a proper release process setup for orbit-electron, so we'll go with version "master" for now. This will change when we get the time to setup versioning in the Orbit's release process. Think of "master" as "latest" or "nightly".